include/cutlass/gemm/collective/xe_mma_w8a8.hpp

```diff
@@ -0,0 +1,281 @@
+  using atom_load_B = Copy_Atom<traits_load_B, ElementB>;
+
+  using TensorMKL = decltype(make_tensor(make_gmem_ptr(static_cast<ElementA const*>(nullptr)), make_shape(0,0,0), StrideA{})); // UINT8
+  using TensorNKL = decltype(make_tensor(make_gmem_ptr(static_cast<ElementB const*>(nullptr)), make_shape(0,0,0), StrideB{})); // UINT8
+
+  using CopyThreadShape = Shape<_1, Int<SubgroupSize>>;
+  using Copy_A = decltype(make_tiled_copy(atom_load_A{},
+                                   Layout<CopyThreadShape>{},
+                                   make_layout(shape_div(typename traits_load_A::BlockShape{}, CopyThreadShape{}))));
+  using Copy_B = decltype(make_tiled_copy(atom_load_B{},
+                                   Layout<CopyThreadShape>{},
+                                   make_layout(shape_div(typename traits_load_B::BlockShape{}, CopyThreadShape{}))));
+  // Host side kernel arguments
+  struct Arguments {
+    ElementA  const* ptr_A;
+    StrideA dA;
+    ElementB const* ptr_B;
+    StrideB dB;
+  };
+
+  struct Params {
+    TensorMKL mA;
+    TensorNKL mB;
+  };
+
+  //
+  // Methods
+  //
+
+  CollectiveMma() = default;
+
+  template <class ProblemShape>
+  static constexpr Params
+  to_underlying_arguments(ProblemShape const& problem_shape, Arguments const& args, void* workspace) {
+    (void) workspace;
+
+    auto [M,N,K,L] = problem_shape;
+    
+    auto mA_mkl = make_tensor(make_gmem_ptr(static_cast<ElementA const*>(args.ptr_A)),
+                              make_layout(make_shape(M, K, L), args.dA));
+    auto mB_nkl = make_tensor(make_gmem_ptr(static_cast<ElementB const*>(args.ptr_B)),
+                              make_layout(make_shape(N, K, L), args.dB));
+    return Params{mA_mkl, mB_nkl};
+  }
+
+  template <class EngineIn,
+      class EngineOut,
+      class LayoutIn,
+      class LayoutOut,
+      class... Ts>
+  CUTLASS_DEVICE
+  void convert_E4M3_to_FP16(
+          Tensor<EngineIn, LayoutIn> const& in,
+          Tensor<EngineOut, LayoutOut>& out) {
+
+      static_assert(is_rmem<EngineIn>::value, "Input tensor for A conversion must come from registers");
+      static_assert(is_rmem<EngineOut>::value, "Output tensor for A conversion must come from registers");
+      static_assert(cosize_v<LayoutIn> == cosize_v<LayoutOut>);
+      static_assert(size_v<LayoutIn> == cosize_v<LayoutIn>);
+      static_assert(size_v<LayoutOut> == cosize_v<LayoutOut>);
+
+      using SrcType = typename EngineIn::value_type;
+      using DstType = typename EngineOut::value_type;
+
+      static_assert(std::is_same_v<SrcType, uint8_t>, "Expected fp8 (E4M3) input as uint8_t");
+      static_assert(std::is_same_v<DstType, half_t>, "Expected fp16 output as half_t");
+
+      auto const& src = in(_, _, _);
+      auto const& dst = out(_, _, _);
+
+      SrcType const* pSrc = src.data();
+      DstType* pDst = dst.data();
+
+      constexpr int num_elements = decltype(size(src))::value;
+      constexpr int vec_size = 16;
+
```

Comment:
Could you add a TODO for us to move this conversion inside the Cutlass conversion mechanism?

```suggestion
TODO(Codeplay): Move conversion to NumericArrayConverter 
```